index.js

```diff
@@ -166,6 +166,56 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Changes the theme based upon the value passed
+ * @param {int}     themeNum - a number passed based upon what theme button the user selected
+ */
+function changeTheme(themeNum) {
+    // set the theme slug based upon the number theme passed
+    if(themeNum == 1)
+        var personaSlug = "fox-in-snow";
+    else if(themeNum == 2)
+        var personaSlug = "cozy-fox";
+    else if(themeNum == 3)
+        var personaSlug = "shiretoko-fox";
```

Comment:
You do not need to declare the variable each time instead, declare it once as the first line of the function, and then simply update it here. So,

```
function changeTheme(themeNum) {
    var personaSlug;
    ....
```